packages/codemod/src/utils/prop-utils.ts

```diff
@@ -15,25 +15,26 @@ import { logPropMigrationError } from "./report-utils";
  * Updates a prop name in a JSX element
  */
 export function updatePropName(
-  j: JSCodeshift,
-  elementPath: ASTPath<JSXElement>,
+  propCollection: Collection<JSXAttribute>,
   propsNamesMappingOldToNew: Record<string, string>
 ): void {
-  j(elementPath)
-    .find(JSXOpeningElement)
-    .find(JSXIdentifier)
-    .forEach(identifierPath => {
-      if (propsNamesMappingOldToNew[identifierPath.node.name]) {
-        identifierPath.node.name = propsNamesMappingOldToNew[identifierPath.node.name];
-      }
-    });
+  propCollection.forEach(attr => {
+    const propName = attr.node.name.name;
+    if (typeof propName !== "string") return;
```

Comment:
we should tackle this one day and check throughly on which cases we don't have `.name.name` as a string (out of scope for current phase IMO)